js/src/dapps/static/console.js

```diff
@@ -17,15 +17,19 @@
 /* eslint-disable */
 // TODO: Fix linting issues
 
-if (typeof(window.parity) == 'object')
-  window.api = window.parent.secureApi;
-  window.parity.api.subscribe('eth_blockNumber', function (error, blockNumber) {
-    if (error) {
-      console.log('error', error);
-      return;
-    }
-    refreshWatches();
-  });
+if (typeof(window.parent.secureApi) == 'object')
+	window.api = window.parent.secureApi;
+else if (typeof(window.parity) == 'object')
+  window.api = window.parity.api;
+
+if (typeof(window.api) === 'object')
+	window.api.subscribe('eth_blockNumber', function (error, blockNumber) {
+	    if (error) {
+	    	console.log('error', error);
+	    	return;
+	    }
+	    refreshWatches();
+	});
```

Comment:
Indentation of this block looks very random, like mixing on tabs/spaces. (Pre-linting, visual check)